.github/workflows/reusable_test_packages.yaml

```diff
@@ -54,7 +57,7 @@ jobs:
       - name: Run tests
         env:
           LSAN_OPTIONS: "intercept_tls_get_addr=0"
-        uses: falcosecurity/testing@main    
+        uses: falcosecurity/testing@32e319ae505fb330ae74db4502e605a5e517ff22 # main    
```

Comment:
Given this is our own repo we actually want to always point to latest main, not the current one. 
Can you revert this line?

Reply:
/hold